src/modules/robot/arm_solutions/RotatableDeltaSolution.h

```diff
@@ -0,0 +1,40 @@
+#ifndef RotatableDeltaSolution_H
+#define RotatableDeltaSolution_H
+#include "libs/Module.h"
+#include "BaseSolution.h"
+
+class Config;
+
+class RotatableDeltaSolution : public BaseSolution {
+    public:
+        RotatableDeltaSolution(Config*);
+        void cartesian_to_actuator( float[], float[] );
+        void actuator_to_cartesian( float[], float[] );
+
+        bool set_optional(const arm_options_t& options);
+        bool get_optional(arm_options_t& options);
+
+    private:
+        void init();
+        int delta_calcAngleYZ(float x0, float y0, float z0, float &theta);
+        int delta_calcForward(float theta1, float theta2, float theta3, float &x0, float &y0, float &z0);
+
+        float delta_e;			// End effector length
+        float delta_f;			// Base length
+        float delta_re;			// Carbon rod length
+        float delta_rf;			// Servo horn length
+        float delta_z_offset ;		// Distance from delta 8mm rod/pulley to table/bed
+			       		// NOTE: For OpenPnP, set the zero to be about 25mm above the bed
+
+        float delta_ee_offs;		// Ball joint plane to bottom of end effector surface
+        float tool_offset;		// Distance between end effector ball joint plane and tip of tool
+        float z_calc_offset;
+
+        float z_home_offs;		// This is calculated from the above angle, after applying forward
+        				// kinematics, and adding the Z calc offset to it.
+
+        float xyz_full_steps_per_rotation;	// stepper motor steps per 360 full rotation
+
+        float delta[3] = {0.0, 0.0, 0.0};
```

Comment:
I cannot see where this is used at all. Please note we are very short on memory so using any memory when it is not required is not good.